perf/benchmarkapps/GrpcClient/grpc-client.yml

```diff
@@ -0,0 +1,18 @@
+﻿jobs:
+  grpc-client:
+    source:
+      repository: https://github.com/jamesnk/grpc-dotnet.git
+      branchOrCommit: jamesnk/interoptestclient
+      project: perf/benchmarkapps/GrpcClient/GrpcClient.csproj
+    readyStateText: gRPC Client
+    runtimeVersion: '5.0.*'
```

Comment:
I see that you copied from the signalr client, but we'll change it eventually as I prefer these to not vary on each run by default, just when specified on the CI runs.

Reply:
So remove runtime version?

Reply:
All the other client yml files are specifying a runtime, e.g. wrk.yml

```
sdkVersion: 3.1.102
aspNetCoreVersion: '3.1'
runtimeVersion: '3.1'
```

Tell me what to put here and I'll change it.

Reply:
These values would be fine